src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -55,6 +55,7 @@ public partial class DynamoModel : INotifyPropertyChanged, IDisposable, IEngineC
         private readonly PathManager pathManager;
         private WorkspaceModel currentWorkspace;
         private Timer backupFilesTimer;
+        private Dictionary<Guid, string> backupFilesDict = new Dictionary<Guid, string>();
```

Comment:
It seems like `backupFilesDict` data member isn't required, its value is always equivalent to that of `PreferenceSettings.BackupFiles`, and if we can read the file list from `PreferenceSettings.BackupFiles` in `OnRequestDispatcherBeginInvoke`, then we won't need it. Does that make sense?